src/FormField/FormField.uni.driver.js

```diff
@@ -48,5 +49,12 @@ export const formFieldUniDriverFactory = (base, body, { dataHook }) => {
       );
       return await testkit.getTooltipText();
     },
+    getSuffix: async () => {
```

Comment:
The better way is to do:
```suggestion
    getSuffix: () => base.$(`[data-hook="${dataHooks.suffix}"]`)
```

However, because we support the legacy driver and it returns the native node - the method needs to be compatible and imitate the legacy as long as we run the same tests using `runTests` approach on both drivers.